File: PHP/palindromes.php
<?php 
$file = fopen('/usr/share/dict/words', 'r'); 
while (!feof($file)) { 
    $line = rtrim(fgets($file)); 
    $lower = lcfirst($line);  
    $reverse = strrev($lower);
    if ($line != NULL && $lower == $reverse) {
    	echo("$line is a palindrome\n"); 
    }
}

File: Julia/palindromes.jl
open("/usr/share/dict/words","r") do file
    for line in eachline(file)
    	line = chomp(line)
    	lower = lcfirst(line)
    	if lower == reverse(lower)
		    println(line, " is a palindrome")
    	end
    end
end
